### PR TITLE
Update names to be more consistent and add aliases

### PR DESCRIPTION
**Pull request by michaeloa**
Summary:
This changes the names of the commands to be more consistent and self-explanatory. The change is backwards compatible (old names are
preserved as aliases). Also updates the documentation and adds aliases to better further improve ease of use.
Issue ID(s): TDX-616

Testing:
- [ ] Unit Tests
- [ ] Integration Tests

Documentation:
- No updates

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -20,4 +20,5 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 
 [{*.py, *.pyi}]
-indent_size = 4
\ No newline at end of file
+indent_size = 4
```
Comment (andeki92) [inline .editorconfig:23]: kinda messy getting both edits in this one PR 😢

File: alias.yml [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
+status: dxp status
+template: dxp template
 test: dxp test
 
-# Pull Requests
-prc: dxp pr
-prm: dxp merge
-prs: pr list
+# More concise PR aliases
+prc: dxp pr-create
```
Comment (andeki92) [inline alias.yml:20]: Might be personal preference, but they way I read this is that we have modifiers of the PR-command here. Meaning i read this as 

<command> <target> <modifier> etc.

In this case i think the list, create, merge etc should be part of the pr subcommand and should therefore be

- dxp pr (doesn't do anything as it has no modifier - maybe help text, w/e)
- dxp pr create
- dxp pr list
- dxp pr merge
- dxp pr view

Another alternative is to do something akin to the kubectl command where you have <command> <modifier> <target> (e.g. kubectl get pods) (i think this works better if you have many targets)

This also makes the aliases flow a bit more smoothly imo - like pr is short for dxp pr, prl is short for pr list etc (making nesting a bit easier)

**Review by andeki92 — APPROVED** [review]
I'll approve it, but would prefer this to be two non-dependent PRs and the comment above to be resolved

File: .editorconfig (hunk 1 above)
Comment (michaeloa) [reply .editorconfig:23]: Yes, this is the problem with how GitHub handles (or pretty much doesn't handle) stacked commits. :(

File: alias.yml (hunk 2 above)
Comment (michaeloa) [reply alias.yml:20]: Hmm... I can buy that. Future PR.

**Pull request merged**